lib/ACL/RuleManager.php

```diff
@@ -293,6 +295,20 @@ public function saveRule(Rule $rule): void {
 				->andWhere($query->expr()->eq('mapping_type', $query->createNamedParameter($rule->getUserMapping()->getType())))
 				->andWhere($query->expr()->eq('mapping_id', $query->createNamedParameter($rule->getUserMapping()->getId())));
 			$query->executeStatement();
+
+			if ($rule->getUserMapping()->getType() === 'user') {
+				$logMessage = 'The ACL rule was updated for file/folder with id: "%s" for the user "%s" with permission "%s" and mask "%s"';
+			} else {
+				$logMessage = 'The ACL rule was updated for file/folder with id: "%s" for the group "%s" with permission "%s" and mask "%s"';
+			}
+			$params = [
+				'fileId' => $rule->getFileId(),
```

Comment:
are we mixing parameter order with named parameters ? with associative arrays I'm not sure if the order is guaranteed
is this how we do it in other places, looks weird